src/main/java/com/r307/arbitrader/service/paper/PaperTradeService.java

```diff
@@ -226,15 +229,21 @@ private void checkBalance(LimitOrder order, BigDecimal averagePrice) {
      * Throw a FundsExceededException if the funds are not sufficient
      */
     private void checkBalance(LimitOrder order) {
```

Comment:
If this checks for funds, for margin enabled, etc. should we rename both checkBalance to verifyOrder ?